index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="icon" type="image/x-icon" href="/assets/favicon_io/favicon-32x32.png">
+    <link rel="stylesheet" href="/css/style.css" />
+    <title>Omnifood</title>
+  </head>
+  <body>
+    <div class="header">
+      <img src="/assets/omnifood-logo.png" alt="img" class="img">
+      <div class="header-right">
```

Comment:
This is not the right approach to creating a navigation bar. Use 'nav' element instead of 'div'. Then use any list element like 'ul' or 'ol' to list those links. Instead of directly using an anchor('a' tag) element use it inside a 'li' element. 
Navigation menu names:
1. How it works
2. Meals
3. Testimonials
4. Pricing
5. Try for free
Change the navigation names as per the above list. The last navigation link is a button.